data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -332,6 +332,9 @@ then
 	if [ -f "$PWD/manifests/cloud-provider-config.yaml" ]; then
 		ADDITIONAL_FLAGS="${ADDITIONAL_FLAGS} --cloud-config-file=/assets/config-bootstrap/cloud-provider-config-generated.yaml"
 	fi
+	if [ -f "/opt/openshift/manifests/openshift-lbconfig-custom-dns.yaml" ]; then
+		ADDITIONAL_FLAGS="${ADDITIONAL_FLAGS} --cloud-lb-config-file=/assets/manifests/openshift-lb-config.yaml"
```

Comment:
Should we make the names match ?
```suggestion
		ADDITIONAL_FLAGS="${ADDITIONAL_FLAGS} --cloud-lb-config-file=/assets/manifests/openshift-lbconfig-custom-dns.yaml"
```

Reply:
We should. As discussed earlier, it would be better if "custom-dns" is not part of the name. We should name the ConfigMap based on its contents. See : https://github.com/openshift/installer/pull/7631/files#r1394420557